### ISSUE TITLE
Crash on startup on jazzy

### ISSUE THREAD

**Issue by ottojo** [ros-visualization/rqt_tf_tree#46]
In ros 2 jazzy, rqt_tf_tree seems to crash on starting, both standalone and as an rqt plugin, with the following error:

```
jonas@ade1 ➜  workspace ros2 run rqt_tf_tree rqt_tf_tree
PluginHandlerDirect._restore_settings() plugin "rqt_tf_tree/RosTfTree#0" raised an exception:
Traceback (most recent call last):
  File "/opt/ros/jazzy/lib/python3.12/site-packages/qt_gui/plugin_handler_direct.py", line 121, in _restore_settings
    self._plugin.restore_settings(plugin_settings_plugin, instance_settings_plugin)
  File "/opt/ros/jazzy/lib/python3.12/site-packages/rqt_tf_tree/tf_tree.py", line 131, in restore_settings
    self._refresh_tf_graph()
  File "/opt/ros/jazzy/lib/python3.12/site-packages/rqt_tf_tree/tf_tree.py", line 143, in _refresh_tf_graph
    self._update_graph_view(self._generate_dotcode())
                            ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/ros/jazzy/lib/python3.12/site-packages/rqt_tf_tree/tf_tree.py", line 153, in _generate_dotcode
    return self.dotcode_generator.generate_dotcode(dotcode_factory=self.dotcode_factory,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/ros/jazzy/lib/python3.12/site-packages/rqt_tf_tree/dotcode_tf.py", line 97, in generate_dotcode
    self.graph = self.generate(data, timer.now().nanoseconds / rclpy.time.CONVERSION_CONSTANT)
                                                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'rclpy.time' has no attribute 'CONVERSION_CONSTANT'

Traceback (most recent call last):
  File "/opt/ros/jazzy/lib/python3.12/site-packages/rqt_tf_tree/tf_tree.py", line 138, in _update_tf_graph
    self._refresh_tf_graph()
  File "/opt/ros/jazzy/lib/python3.12/site-packages/rqt_tf_tree/tf_tree.py", line 143, in _refresh_tf_graph
    self._update_graph_view(self._generate_dotcode())
                            ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/ros/jazzy/lib/python3.12/site-packages/rqt_tf_tree/tf_tree.py", line 153, in _generate_dotcode
    return self.dotcode_generator.generate_dotcode(dotcode_factory=self.dotcode_factory,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/ros/jazzy/lib/python3.12/site-packages/rqt_tf_tree/dotcode_tf.py", line 97, in generate_dotcode
    self.graph = self.generate(data, timer.now().nanoseconds / rclpy.time.CONVERSION_CONSTANT)
                                                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'rclpy.time' has no attribute 'CONVERSION_CONSTANT'
[ros2run]: Aborted
```

**Comment by ottojo** [comment]
This seems to be related to the following rclpy change: https://github.com/ros2/rclpy/commit/6f2507f9325054cd38a33424ed7ce55de327004b

**Comment by arne48** [comment]
This is now fixed by #48
